core-web/libs/edit-content/src/lib/fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
+
+import { ControlContainer, FormGroupDirective } from '@angular/forms';
+
+import { DotEditContentRadioFieldComponent } from './dot-edit-content-radio-field.component';
+
+import {
+    RADIO_FIELD_BOOLEAN_MOCK,
+    RADIO_FIELD_FLOAT_MOCK,
+    RADIO_FIELD_INTEGER_MOCK,
+    RADIO_FIELD_TEXT_MOCK,
+    createFormGroupDirectiveMock
+} from '../../utils/mocks';
+
+describe('DotEditContentRadioFieldComponent', () => {
+    let spectator: Spectator<DotEditContentRadioFieldComponent>;
+
+    const createComponent = createComponentFactory({
+        component: DotEditContentRadioFieldComponent,
+        componentViewProviders: [
+            { provide: ControlContainer, useValue: createFormGroupDirectiveMock() }
+        ],
+        providers: [FormGroupDirective],
+        detectChanges: false
+    });
+
+    beforeEach(() => {
+        spectator = createComponent();
+    });
+    describe('DotEditContentRadioFieldComponent with Text DataType', () => {
```

Comment:
We already have `DotEditContentRadioFieldComponent` in the first describe, don't need to added here because in the results looks like redundant:

<img width="790" alt="image" src="https://github.com/dotCMS/core/assets/751424/420c9066-6ef2-4f92-8220-89c286f467f7">